packages/integrations/image/README.md

```diff
@@ -341,6 +341,22 @@ import heroImage from '../assets/hero.png';
 <Image src={import('../assets/hero.png')} />
 ```
 
+#### Images in `/public`
+
+Files in the `/public` directory are always served or copied as-is, without transform or bundling. This also means that files in `/public` also can't be `import`ed directly. It's recommended that local images are always kept in `/src`, but in case you absolutely need to keep an image in `/public` it can be referenced by path and treated as a remote image.
+
+```astro
+---
+import { Image } from '@astrojs/image/components';
+import socialImage from '/social.png';
+---
+
+// the image file from `/public/social.png` will be used
+// in static (SSG) builds, this will be built and optimized to `/dist`.
+// in SSR builds, the image will be optimized by the server when requested by a browser.
+<Image src={socialImage} width={1280} aspectRatio="16:9" />
+```
```

Comment:
Just checking that I'm interpreting this correctly:

When you say "can't be imported directly" do you mean inside the component itself as a `src` attribute? Because, it *is* imported into frontmatter, right? Assuming that's the case... Not sure this wording is perfect, so go ahead and edit, but I would massage it a little bit. (And, I gave in and channeled my inner Ben to add "like so"...)

``````suggestion
Files in the `/public` directory are always served or copied as-is, with no processing. We recommend that local images are always kept in `src/` so that Astro can transform, optimize and bundle them. But if you absolutely must keep an image in `public/`, use its relative URL path as the image's `src=" attribute. It will be treated as a remote image, which requires an `aspectRatio` attribute.

Alternatively, you can import an image from your `public/` directory in your frontmatter and use a variable in your `src=` attribute. You cannot, however, import this directly inside the component as its `src` value. 

For example, use an image located at `public/social.png` in either static or SSR builds like so:

```astro title="src/pages/page.astro"
---
import { Image } from '@astrojs/image/components';
import socialImage from '/social.png'; 
---
// In static builds: the image will be built and optimized to `/dist`.
// In SSR builds: the image will be optimized by the server when requested by a browser.
<Image src={socialImage} width={1280} aspectRatio="16:9" />
```
``````